ieee.csl

```diff
@@ -342,6 +372,15 @@
             <text variable="number"/>
             <text macro="issued"/>
           </group>
+        </else-if>
+		<!-- Online Video -->
+        <else-if type="motion_picture">
+          <text macro="location" suffix=". "/>
```

Comment:
I don't quite understand this -- what do you envision "location" is in the example citations? I'm reading the sample ctitations as author (date) "title" [medium], Available...

Reply:
It's a physical location:

Video Owner/Creator, Location (if available). Title of Video: In Initial Caps. (Release date). Accessed: 
Month Day, Year. [Online Video]. Available: http://URL.onlinevideo.org

![Screenshot_20210424-141627_Drive.jpg](https://user-images.githubusercontent.com/43231870/115946933-d9102480-a507-11eb-8351-87895c8a88eb.jpg)

Reply:
Changed to geographic-location